src/Object/Api/Mastodon/Status.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Friendica\Object\Api\Mastodon;
+
+use Friendica\BaseEntity;
+
+/**
+ * Class Relationship
+ *
+ * @see https://docs.joinmastodon.org/entities/status/
+ */
+class Status extends BaseEntity
```

Comment:
Was this class intentionally added? Tbh it seems like a crossover of another feature branch :)

Reply:
Ugh, you're right, it was in a stash that got automatically staged for some reason?